### PR TITLE
Add debiased PSF moments to Sourcetable

### PR DESCRIPTION
**Pull request by arunkannawadi**

### CODE REVIEW

File: schemas/Source.yaml
Comment (yalsayyad) [reply schemas/Source.yaml:395]: Posting only because I had to look it up again:
```
'slot_PsfShape'->'ext_shapeHSM_HsmPsfMoments'
```

Comment (arunkannawadi) [reply schemas/Source.yaml:395]: Do you mean make `ixxPSF` be `ext_shapeHSM_HsmPsfMoments_xx` explicitly?

Comment (arunkannawadi) [reply schemas/Source.yaml:395]: Looking back at the Object.yaml, `iDebiasedPSF_flag` looks like a bad naming, since `i` there refers to `i` band and is not the same as the `i` in `ixx` etc. I am more inclined to make the flag names in Object.yaml more meaningful. For e.g., `i_flag` doesn't indicate that it's the object's second-order moments that is flagged. Similarly with `iPSF_flag`.

Comment (jmeyers314) [reply schemas/Source.yaml:395]: But I'll leave that up to @yalsayyad .  (And maybe a separate ticket).
Other than this, this all looks good to me.

**Review by jmeyers314 — APPROVED** [review]

File: schemas/Source.yaml
Comment (arunkannawadi) [reply schemas/Source.yaml:395]: Oh, changing all the occurrences of `ixx` -> `Ixx` etc. is going to be a much bigger pain. I'm torn between maintaining consistency between Source and Object tables and making the flag names more meaningful. I'm going to incorporate @yalsayyad 's suggested changes for now, but will wait to hear back before merging.

Comment (yalsayyad) [reply schemas/Source.yaml:395]: Sorry for confusing everyone. `'slot_PsfShape'->'ext_shapeHSM_HsmPsfMoments'`  wasn't a comment, suggestion or request. It was just informational if anyone was wondering what slot_PsfShape in Source Tables were like I was.

**Pull request merged**
